src/universal/components/SignInPage/SignInEmailPasswordForm.js

```diff
@@ -36,21 +40,27 @@ const SignInEmailPasswordForm = (props: Props) => (
   <Form onSubmit={props.handleSubmit}>
     <FieldsContainer>
       <Field
+        type="email"
         autoFocus
         component={InputField}
         placeholder="you@company.co"
         label="Email:"
         name="email"
+        underline
+        disabled={props.submitting}
       />
       <Field
+        type="password"
         component={InputField}
         placeholder="********"
         label="Password:"
         name="password"
+        underline
+        disabled={props.submitting}
       />
     </FieldsContainer>
     <Button
-      disabled={!props.valid}
```

Comment:
perhaps make the button `disabled` when they fail validation & make it `waiting` when waiting for a response from the server?

Reply:
Yes! Oops :)